docs/drafts/software-development-maintenance-plan.md

```diff
@@ -175,29 +149,29 @@ All public functions will be documented through doc strings or comments.
 
 For each commit there is a pre-commit check which verifies that:
 
-- All code ad-hears to the black style guide
+- All code adheres to the black style guide
 - Imports are appropriately organised
 - Jupyter notebook outputs are not committed
 - TOML files are valid
 
 For each pull request, there are automated checks within GitHub actions which
-check the items described within **8 Software System Test Activities**.
+check the items described within [](software-tests).
 
 Either manual testing is undergone or new automated tests are written to verify
 that the current code changes fulfil the software requirements.
 
-Should the team be a size larger than 1 then all PRs are to undergo code review
-before merging.
+All PRs are to undergo code review before merging.
+
+(software-tests)=
 
 ## 8 Software System Test Activities
 
 For every pull request the following automated GitHub actions based system
 tests are undergone:
 
-- Python and Typescript unit and integration testing suites
-- Python and Typescript linting
-- Typescript type checker
-- Python mypy type checker
+- Python unit and integration testing suites with pytest
+- Python linting with pylint
+- Python pyright type checker
```

Comment:
You mentioned `pylance` earlier in addition to `pyright`
you don't mention `pylance` here

Reply:
Ah yes. They're actually the same tool (sort of). Pylance is VSCode's utilisation of pyright with a few extra additions. When it is called from the command line one uses pyright directly (and can't use pylance in the CI suite). When in use within vscode one is using pyright via pylance.

https://github.com/microsoft/pylance-release/blob/main/FAQ.md